### PR TITLE
Bluetooth: controller: Map debug pins to P3 pin head on nRF5x DK

### PR DESCRIPTION
**Pull request by cvinayak**
Updated debug pin mapping so that the outputs are on P3 pin
head on all nRF5x Development Kits.

Signed-off-by: Vinayak Kariappa Chettimada <vich@nordicsemi.no>

### CODE REVIEW

File: subsys/bluetooth/controller/hal/nrf5/debug.h [hunk 1]
```diff
@@ -8,157 +8,202 @@
...
+#define DEBUG_PIN2       BIT(14)
+#define DEBUG_PIN3       BIT(15)
+#define DEBUG_PIN4       BIT(16)
+#define DEBUG_PIN5       BIT(17)
+#define DEBUG_PIN6       BIT(18)
+#define DEBUG_PIN7       BIT(19)
+#define DEBUG_PIN8       BIT(20)
+#define DEBUG_PIN9       BIT(23)
+#else
+#error "Unknown nRF5x board."
```
Comment (carlescufi) [inline subsys/bluetooth/controller/hal/nrf5/debug.h]: Better have something like: 
"BLUETOOTH_CONTROLLER_DEBUG_PINS not supported on this board, please add an entry in hal/nrf5/debug.h"
or something similar?

Comment (cvinayak) [inline subsys/bluetooth/controller/hal/nrf5/debug.h]: Sure. Thanks.

**Review by carlescufi — DISMISSED** [review]

**Comment by carlescufi** [comment]
@cvinayak can you rebase this and fix the BLUETOOTH_ into BT_ ?

**Comment by cvinayak** [comment]
@carlescufi, done, rebased to BT_CTLR changes in master

**Comment by cvinayak** [comment]
@carlescufi ping!

**Review by carlescufi — APPROVED** [review]

**Pull request merged**